samples/servicecontrol/adapter-rabbitmq-different-topologies/sample.md

```diff
@@ -33,12 +33,12 @@ NOTE: In order to connect to a different RabbitMQ broker, ensure all connection
 
 ## Running the project
 
- 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure adapter starts first because on start-up it creates a queue that is used for heartbeats.
+ 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure the adapter starts first because on start-up it creates a queue that is used for heartbeats.
```

Comment:
I believe the issue is that with "just hit F5" you can't guarantee the order in which projects are started, which may look as the sample is not working correctly.